packages/api/src/entity/newsletter_email.ts

```diff
@@ -34,4 +34,13 @@ export class NewsletterEmail {
 
   @OneToMany(() => Subscription, (subscription) => subscription.newsletterEmail)
   subscriptions!: Subscription[]
+
+  @Column('text')
```

Comment:
I'd maybe argue this could be null and we'd fallback in code, but probably either way is the same.

Reply:
i guess we need it because the existing newsletters have to be saved in `inbox` and newer ones in `following`